lms/templates/credit/fake_credit_eligibility_status.html

```diff
@@ -0,0 +1,27 @@
+<%namespace name='static' file='/static_content.html'/>
+<html>
+<head><title>Fake Page to Update the minimum grade course requirement status</title>
+</head>
+<body>
+<p>Fake Credit Eligibility
+<div>
+% if error:
+    <div style="padding-top: 5px;color: red">${ error }</div>
```

Comment:
As discussed in hipchat on 9th June @awais786 to update to move to a style block in the head of the page. Please let me know when this PR has been updated to achieve this @awais786.